index.bs

```diff
@@ -2400,6 +2397,9 @@ error.
     the value of |rpId| for all client processing should be replaced by the
     value of |appid|.
 
+: Client data
+:: Returns the JSON value `true` to indicate to the RP that the extension was acted upon
+
```

Comment:
Might this be clearer if the six standard sections had these titles, rather than the current ones?
 - Client input data
 - Client processing
 - Client output data
 - Authenticator input data
 - Authenticator processing
 - Authenticator output data

What you're mostly talking about I think is client processing, rather than the client output data.  Of course, you could be proposing that a particular extension, the "appid" extension, have a different return value than "true".  That's fine with me.  If so, make a proposal.